qemu/tests/cfg/win_nics_teaming.cfg

```diff
@@ -0,0 +1,14 @@
+- win_nics_teaming:
+    virt_test_type = qemu
+    only Windows
```

Comment:
only support windows server according to case steps, so , we should limit the guest os

Reply:
I suggest using `only Win2012.x86_64.r1 Win2012.x86_64.r2 Win2016.x86_64 Win2019.x86_64`

Reply:
But note that if it comes out new server version, we should add this to the cfg.

Reply:
I will change it when next update.